setup/updating.md

```diff
@@ -65,8 +65,43 @@ To update Coder, follow these steps:
    ```console
    helm upgrade --namespace coder --install --atomic --wait \
      --version 1.16.1 coder coder/coder --values current-values.yaml
+
+   ```
+
+1. [Update your Helm chart](../guides/admin/helm-charts.md) as follows:
+
+   Rename all instances of `cemanager` to `coderd`.
+
+   Remove all `dashboard.*` values. Route any traffic to `coderd`.
+
+   Remove all `envproxy.*` values. Route any traffic to `coderd`.
+
+   Redirect all traffic from your custom ingress to `coderd`.
+
+   `coderd` now serves TLS certificates; to upgrade the NGINX ingress
+   controller, set `coderd.serviceNext` to `true`. (See our
+   [TLS certificates](../guides/tls-certificates/index.md) for more information
+   on how to configure Coder to issue and use certificates.)
+
+   Move TCP port `8080` to `80`.
+
+   Move TCP port `8443` to `443`.
+
+1. Update Coder with your new Helm chart values:
+
+   ```console
+   helm upgrade coder coder/coder -n coder --version=<VERSION> --values current-values.yaml
```

Comment:
We should clarify that updating the helm chart is only need when upgrading from `1.20` to `1.21`.